modules/glean/runtime/glean-plugin.client.ts

```diff
@@ -11,18 +13,67 @@ export default defineNuxtPlugin((nuxtApp) => {
     const devMode = useAppConfig().env === ('dev' || 'canary' || 'preview')
     const uploadEnabled = devMode // this will eventually be a setting that the user can toggle
 
-    Glean.initialize(GLEAN_APP_ID, uploadEnabled, { })
+    Glean.initialize(GLEAN_APP_ID, uploadEnabled, {})
```

Comment:
@wtfluckey you might want to provide additional information here in the configuration, for example the version and/or channel ;)